examples/librimix/librimix.ipynb

```diff
@@ -84,7 +84,7 @@
     "for partition, manifests in librimix_manifests.items():\n",
     "    feature_set_builder = FeatureSetBuilder(\n",
     "        feature_extractor=FeatureExtractor(type='fbank'),\n",
-    "        output_dir=f'librimix_nb/feats_{partition}',\n",
+    "        output_dir='{}/librimix_nb/feats_{}'.format(root_dir, partition),\n",
```

Comment:
Mm, it might be worthwhile considering the case where root_dir is in a shared place not owned by this user, and they want to dump feats to somewhere else.

Reply:
I think it should be already possible.

Reply:
The bash scripts embedded in Librimix's ipynb was rewrote in Python. And it allowed to set output directory like MiniLibriSpeech's.